tests/playwright-tests/README.md

```diff
@@ -62,6 +62,11 @@ npx playwright test
 npx playwright test <spec-filename.ts>
 ```
 
+### Running the test in a folder level
+```bash
+npx playwright test --project "projectname"
```

Comment:
Should this heading read "Running the test in a folder level" or "Running the test in a project level".

If "Running the test in a folder level", the `npx playwright test --project "projectname"` guidance text should probably read `npx playwright test --project "foldername"` instead.